### PR TITLE
Replace blacklist term in Grafana Dashboard

### PR DESCRIPTION
**Pull request by ChrisKujawa**
## Description

Replace the blacklist term with a non-racial loaded term. Updated the grafana dashboard and adjusted the query such it will return for the old and new metrics values.

### CODE REVIEW

File: monitor/grafana/zeebe.json [hunk 1]
```diff
@@ -503,13 +503,13 @@
                 "uid": "${DS_PROMETHEUS}"
               },
               "editorMode": "code",
-              "expr": "max(zeebe_blacklisted_instances_total{namespace=~\"$namespace\", pod=~\"$pod\", partition=~\"$partition\"}) by (namespace)",
+              "expr": "max(zeebe_blacklisted_instances_total{namespace=~\"$namespace\", pod=~\"$pod\", partition=~\"$partition\"}) by (namespace) \nor max(zeebe_banned_instances_total{namespace=~\"$namespace\", pod=~\"$pod\", partition=~\"$partition\"}) by (namespace)",
```
Comment (oleschoenburg) [inline monitor/grafana/zeebe.json:506]: :bulb:  TIL about `or`. If I understand correctly, it matches two "vectors" based on the labels. For us that means on the namespace label. If a cluster is updated to a version which uses the new naming, the old time series does not receive new data and we will use the new time series instead. Makes sense :+1:

**Review by lenaschoenburg — COMMENTED** [review]
Thanks @Zelldon :+1: 

There are more panels that need adjustment though, for example "Blacklisted instances by partition"

**Comment by Zelldon** [comment]
Ups I missed this one:

> There are more panels that need adjustment though, for example "Blacklisted instances by partition"

**Comment by Zelldon** [comment]
@oleschoenburg done :white_check_mark:

**Review by lenaschoenburg — APPROVED** [review]
:+1: 

:wrench: Some panel titles and descriptions are still using the old term, just CTRL-F for them.

**Comment by Zelldon** [comment]
:facepalm: Thanks Ole I guess it was too late for me yesterday, sorry for the noise. Done now. Thanks for the review.

**Pull request merged**